workflows/gitops-promotion/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,461 @@
+          - name: git-auth-secret     # Name of a secret with the following keys: token, username, email
+          - name: git-checkout-branch # Optional, branch for changes. Use if you'll be creating a PR.
+            value: main
+          - name: git-clone-branch    # Optional, branch to clone. Match with git-checkout-branch to add to an existing branch/PR.
+            value: main
+          - name: git-commit-msg      # Optional, custom commit message. If not provided, one will be generated.
+            value: ""
+          - name: create-github-pr    # Optional, set to true (and git-checkout-branch to non-main) to create a PR
+            value: false
+          - name: pr-title            # Optional, will use the commit message for PR title if one is not provided
+            value: ""
+          # ------------------------------------------------------------------------------
+          # Strings to replace within the source/dest patterns
+          # ------------------------------------------------------------------------------
+          - name: env-src             # Replaces [[ENV]] in source paths
+          - name: env-dest            # Replaces [[ENV]] in destination paths
+          - name: svc-name-list       # Space-separated list of microservices to promote. Each one
+                                      # replaces [[SVC_NAME]] in paths.
+          - name: other               # Optional, replaces [[OTHER]] in all paths
+            value: ""
+          # ------------------------------------------------------------------------------
+          # Source/dest patterns, where [[ENV]] differentiates btw source and dest
+          # ------------------------------------------------------------------------------
+          - name: file-path-pattern   # Path to the source/destination YAML file
+                                      # kustomization.yaml example: "k8s-resources/[[SVC_NAME]]/overlays/[[ENV]]/kustomization.yaml"
+                                      # Helm values.yaml example: "k8s-resources/[[ENV]]/[[SVC_NAME]]/values.yaml"
+          - name: promotion-type      # Must be one of: kustomize-image, helm-value, helm-dependency, yaml-key
+          - name: kust-image-pattern  # For kustomize-image - name of the image transformer to copy from source to dest
+            value: "[[SVC_NAME]]"
+          - name: yaml-key-pattern    # For helm-value and yam-key - YAML key pattern within values.yaml to copy from source to dest
+            value: ".[[SVC_NAME]].image.tag"
+          - name: helm-dep-pattern    # For helm-dependency - name of the subchart/dependency to copy from source to dest
+            value: "[[SVC_NAME]]"
+      outputs:
+        parameters:
+          - name: codefresh-io-pr-url
+            valueFrom:
+              default: ""
+              path: /tmp/pr-url.txt
+      container:
+        image: quay.io/codefreshplugins/argo-hub-workflows-gitops-promotion-versions-0.0.1-images-gitops-promotion:main
```

Comment:
this image should be public

Reply:
I'll ask DevOps to make it public as soon as this merges.